apps/react/src/features/auth/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,11 @@
+import { memo, FC } from 'react';
+
+import { LoginForm } from '../../components/LoginForm/LoginForm';
+
+const LoginPageComponent: FC = () => (
+  <>
+    <LoginForm />
+  </>
```

Comment:
No need for `Fragment`(`<>`) here.
```suggestion
  <LoginForm />
```